tests/migrations/test_operations.py

```diff
@@ -1972,6 +2019,99 @@ def test_remove_partial_unique_constraint(self):
             'name': 'test_constraint_pony_pink_for_weight_gt_5_uniq',
         })
 
+    def test_add_deferrable_unique_constraint(self):
+        project_state = self.set_up_test_model('test_adddeferrableuniqueconstraint')
+        deferrable_unique_constraint = models.UniqueConstraint(
+            fields=['pink'],
+            name='test_constraint_pony_pink_deferrable',
```

Comment:
Please use shorter name (less than 30 chars), e.g. `deferrable_pink_constraint` (the same in all new tests).